libs/datamodel/core/src/transform/ast_to_dml/db/attributes.rs

```diff
@@ -84,14 +106,39 @@ fn visit_scalar_field_attributes<'ast>(
     ast_model: &'ast ast::Model,
     ast_field: &'ast ast::Field,
     model_data: &mut ModelData<'ast>,
-    scalar_field_data: &mut ScalarField,
+    scalar_field_data: &mut ScalarField<'ast>,
     ctx: &mut Context<'ast>,
 ) {
     ctx.visit_scalar_field_attributes(model_id, field_id, scalar_field_data.r#type, |attributes, ctx| {
         // @map
          attributes.visit_optional_single("map", ctx, |map_args, ctx| {
-             visit_map(map_args, MappedName::Field(model_id, field_id), ctx)
-         });
+             let mapped_name = match visit_map(map_args, ctx) {
+                Some(name) => name,
+                None => return
+             };
+
+                scalar_field_data.mapped_name = Some(mapped_name);
```

Comment:
indentation?

Reply:
ugh, something is wrong with rustfmt in these files